roles/obol_charon/tasks/build_binary.yml

```diff
@@ -7,7 +7,7 @@
     force: true
 
 - name: Build the obol charon binary go build
-  ansible.builtin.command: go build -o charon .
```

Comment:
Just noticed the . after go build -o charon, this will make it use the current directory, maybe this is the issue ?